vesc_driver/src/vesc_interface.cpp

```diff
@@ -201,7 +223,7 @@ void VescInterface::connect(const std::string & port)
         boost::asio::serial_port::stop_bits::one));
   } catch (const std::exception & e) {
     std::stringstream ss;
-    ss << "Failed to open the serial port to the VESC. " << e.what();
+    ss << "Failed to open the serial port " << port << " to the VESC. " << e.what();
```

Comment:
Please remove commented-out code.

Reply:
if I remove the part of the code you have selected, in case of wrong port, I have no evidence of it

Reply:
i just ran a test to check and in case the port is wrong the error is
`
[vesc_driver]: Failed to connect to the VESC, SerialException Failed to open the serial port / tmp/ttyV0 to the VESC. open: No such file or directory failed ..
`
if i remove `<< port << ` it become
`
[vesc_driver]: Failed to connect to the VESC, SerialException Failed to open the serial port to the VESC. open: No such file or directory failed ..
`

you are sure it is okay to delete it?

Reply:
Sorry, put this on the wrong line. This is fine.